contracts/MetadataViews.cdc

```diff
@@ -129,4 +131,71 @@ pub contract MetadataViews {
             return "ipfs://".concat(self.cid)
         }
     }
+
+    /*
+    *  Royalty Views
+    *  Defines the composable royalty standard that gives marketplaces a unified interface
+    *  to support NFT royalties.
+    *
+    *  Marketplaces can query this `Royalties` struct from NFTs 
+    *  and are expected to pay royalties based on these specifications.
+    *
+    */
+
+    /// Interface to provide details of the royalty.
+    pub struct Royalties {
+
+        /// Array that tracks the individual royalties
+        access(self) let cutInfos: [Royalty]
+
+        pub init(cutInfos: [Royalty]) {
```

Comment:
personally I do not see much use for labels in this case when there is a single paramter and it is very obvious what it is.

Reply:
Good point. I made it optional